R2API/ScriptableObjects/ArtifactCode.cs

```diff
@@ -19,11 +21,14 @@ public class ArtifactCode : ScriptableObject {
         private SHA256 hasher;
 
         /// <summary>
-        /// The Sha256HashAsset stored in this scriptable object.
+        /// The Sha256HashAsset stored in this Scriptable Object.
         /// </summary>
-        [HideInInspector]
+        [HideInInspector()]
```

Comment:
The attribute ctor is automatically called with no arguments if you don't add (). Was this needed?

Reply:
didnt know that, will keep it in mind.